src/Monticello/MonticelloBootstrap.class.st

```diff
@@ -60,7 +60,10 @@ MonticelloBootstrap >> bootstrapVersionsSatisfying: aBlock [
 
 	^ RPackageOrganizer default packages
 		select: [ :package | package name ~= RPackage defaultPackageName and: [ aBlock value: package ] ]
-		thenCollect: [ :rpackage | self fileNameForPackageNamed: rpackage name ]
+		thenCollect: [ :rpackage | "Hack, I don't know where this was added the wrong way"
+		rpackage name = #'zinc-resource-meta-core'
```

Comment:
Why is this "hack" here?

Reply:
Because we have to check if the name of the package is in lower cases instead of having upper cases at the start of the words.
This allowed it to work.
Now, is it because the new string primitive behaves in a different way ?
I don't know, this has to be investigated.
Having most pharo tests passing though, helps think that it may not be that, but then, why does it currently work? 

Btw pablo was thinking that this was because someone has used as a protocol name 
'*zinc-resource-meta-core'
instead of:
'*Zinc-Resource-Meta-Core'